package.json

```diff
@@ -7,7 +7,7 @@
   },
   "scripts": {
     "test": "shell=$(basename -- $(ps -o comm= $(ps -o ppid= -p $PPID)) | sed 's/^-//'); make test-$shell",
-    "test/fast": "shell=$(basename -- $(ps -o comm= $(ps -o ppid= -p $PPID)) | sed 's/^-//'); make TEST_SUITE=fast test-$shell",
+    "test/fast": "shell=$(basename -- $(ps -o comm= $(ps -o ppid= -p $PPID)) | sed 's/^-//'); env -i TERM=\"$TERM\" /bin/bash -lc \"make TEST_SUITE=fast test-$shell\"",
```

Comment:
can you help me understand these changes? `env` i get, you're setting up the `$TERM` variable. What does `-lc` do? I see `/bin/bash`, what happens if that's not the path to bash, and what about all the other shells (zsh, sh, dash, ksh)?

Reply:
I'll break down my thinking:
* `env -i` runs the following in a fully cleared environment
* `TERM="$TERM"` brings `$TERM` into that environment
* `/bin/bash -lc` runs the make command in a _login_ (`-l`) shell which first populates the environment by executing various profile files. I figure that running from a login shell is more \~robust~ than running from a blank environment, though I'm definitely not sure. When I run the same command without `-l` the tests all pass but the git checks at the start fail due to a path issue.

Using `/bin/bash` was an oversight. Both `bash` and `sh` work, and I'll make a change to use `bash` instead (since `sh` doesn't list `-l` as an option in the man page) unless you prefer otherwise.

The shell urchin uses to run the tests is taken from the make target `test-$shell`, so the shell used to run `make` doesn't matter.